### PR TITLE
feat: load current block at space level

### LINKED ISSUE #729: chore: remove global loading
We should remove the global loading, the loading is only here to load latest block on each chain, instead we should only load latest block once we load a space and only for the network of the space.

### PR DESCRIPTION
**Pull request by bonustrack**
Closes #729 

This PR remove the global loading to get all networks current blocks and instead load current block for a single network when a space or a proposal is loaded. Also this add auto-import for the stores.

### CODE REVIEW

**Review by bitcoinfinancier — APPROVED** [review]

File: src/App.vue [hunk 1]
```diff
@@ -68,15 +63,3 @@ watch(route, () => {
...
-.backdrop {
-  position: fixed;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
-  z-index: 99;
-  background: rgba(0, 0, 0, 0.4);
-}
-</style>
```
Comment (Sekhmet) [inline src/App.vue:82]: Why was this removed? Without this we won't have backdrop when opening sidebar. Is this intentional?

**Review by Sekhmet — COMMENTED** [review]
One comment, otherwise looks good to me.

File: src/App.vue (hunk 1 above)
Comment (bonustrack) [reply src/App.vue:82]: Good catch, I was assuming it was a duplicate code with the modal.
